accounts-db/src/accounts_db.rs

```diff
@@ -10659,6 +10654,12 @@ pub mod tests {
         );
     }
 
+    #[test]
+    fn test_accountsdb_scan_account_storage_no_bank() {
+        accountsdb_scan_account_storage_no_bank_test_case::<WritableAppendVecProvider>();
+        accountsdb_scan_account_storage_no_bank_test_case::<WritableHotStorageProvider>();
+    }
+
```

Comment:
@brooksprumo: Just want to get quick feedback from you.  Here's the way that I am currently developing to enable accounts-db tests for both AppendVec and TieredStorage.  Does it look good to you?

Reply:
Neat!  This is great!  Thanks for the pointer!

Reply:
Changed the test to use test_case